Converter/Calculator/Program.cs

```diff
@@ -0,0 +1,101 @@
+﻿    using System.Threading.Channels;
+    using Calculator;
+
+    bool c;
+    int b;
+    do
+    {
+        Console.Write("\n Write a number to convert: ");
+        c = int.TryParse(Console.ReadLine(), out b);
+        
+    } while (b < 0 || c ==false);
+
+    
+    Calculator.Converter.ConvertTo(b);
```

Comment:
IT would be better to organize your menu here in Main and use your converter just for converting numbers. But as a result, you have converter which is in charge of both menu and conversion.